include/zephyr/drivers/adc/adc_nrfx.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_ADC_ADC_NRFX_H_
```

Comment:
I suggest to use `adc_nrf_*` namespace (for file and function names) to not imply any specific implementation underneath vendor-specific API.